src/tests/ftest/scripts/main.sh

```diff
@@ -32,6 +44,11 @@ else
     cd "$DAOS_BASE"
 fi
 
+# Copy the pydaos source locally and install it.
+cp -a "$PREFIX"/lib/daos/python pydaos
+pip install ./pydaos
+rm -rf pydaos
```

Comment:
This is something we'll need to make sure the test team, and others using ftest do upfront, right?

Reply:
Eventually.  With this PR probably not as the existing process should work but yes, once this has landed then updating the manual process to reflect this will be required.  The next work in this series will be to update from 82 to 92 version and that will *require* this new process.

Reply:
I do want to give this PR a try locally and on other systems where we run just to make sure we don't accidentally break some assumption. It's difficult enough to get ftest working, and even harder to debug when it stops